### PR TITLE
Add Ash

### PR DESCRIPTION
**Pull request by zachdaniel**

### CODE REVIEW

**Comment by simonewebdesign** [comment]
could you add a dot at the end please?

File: README.md [hunk 1]
```diff
@@ -832,6 +832,7 @@ There are [other sites with curated lists of elixir packages](#other-awesome-lis
 * [RIG](https://github.com/Accenture/reactive-interaction-gateway) - Create low-latency, interactive user experiences for stateless microservices.
 * [sugar](https://github.com/sugar-framework/sugar) - Modular web framework for Elixir.
 * [trot](https://github.com/hexedpackets/trot) - An Elixir web micro-framework.
+* [Ash Framework](https://github.com/ash-project/ash) - A declarative, resource-oriented application framework for Elixir
```
Comment (zachdaniel) [inline README.md:835]:
```suggestion
* [Ash Framework](https://github.com/ash-project/ash) - A declarative, resource-oriented application framework for Elixir.
```

**Comment by zachdaniel** [comment]
Done :)

**Comment by simonewebdesign** [comment]
Thanks!

**Pull request merged**